src/enum.cr

```diff
@@ -275,8 +275,16 @@ struct Enum
   end
 
   # Returns a hash value. This is the hash of the underlying value.
+  # Enum hash is exception from random seeding, cause several Hashes are
+  # initialized on very early stages of bootstrap, and StdHasher seed is not
+  # filled yet.
```

Comment:
Sorry I'm not too convenient in Crystal.
Issues are cause of:
- `enum Signal` used in `src/event/signal_handler.cr`
- `enum LLVM::Attribute` defined in `src/llvm/events/cr` (don't know, where it is used)
- `class Thread` is also used very early in `src/thread.cr` : `@@threads = Set(Thread).new`

If you point me how to reorder prelude to fill `StdHasher` `@@seed` before this hashes constructed, it will be great.

It could be fixed also with defining special-cased Hash, if `def hashkey(key); key.hash; end` will be defined, and then overloaded in subclass specially for this use cases.